nc/admin.py

```diff
@@ -1,16 +1,18 @@
-from ckeditor.widgets import CKEditorWidget
 from django import forms
 from django.contrib import admin
+from django_ckeditor_5.widgets import CKEditor5Widget
```

Comment:
re: your comment about ckeditor, does using django_ckeditor_5 fix everything or do we still get a warning?